test/blocks/global-footer/global-footer.test.js

```diff
@@ -0,0 +1,266 @@
+import { expect } from '@esm-bundle/chai';
+import sinon, { stub } from 'sinon';
+import {
+  allElementsVisible,
+  containerSelector,
+  createFullGlobalFooter,
+  featuredProductsSelectors,
+  insertDummyElementOnTop,
+  legalSelectors,
+  menuSelectors,
+  regionPickerSelectors,
+  selectors,
+  socialLinksSelectors,
+  waitForFooterToDecorate,
+} from './test-utilities.js';
+import baseFooter from './mocks/base-footer.js';
+import fetchedFooter from './mocks/fetched-footer.js';
+import icons from './mocks/icons.js';
+import { mockRes } from '../global-navigation/test-utilities.js';
+import placeholders from './mocks/placeholders.js';
+
+const originalFetch = window.fetch;
+const allSelectorsKeys = Object.keys(selectors);
+
+describe('global footer', () => {
+  let clock = null;
+  beforeEach(async () => {
+    document.body.innerHTML = baseFooter;
+    clock = sinon.useFakeTimers({
+      toFake: ['setTimeout'],
+      shouldAdvanceTime: true,
+    });
+
+    window.fetch = stub().callsFake((url) => {
+      if (url.includes('/footer')) {
+        return mockRes({ payload: fetchedFooter({ regionPickerHash: '/fragments/regions#langnav' }) });
+      }
+
+      if (url.includes('/placeholders')) {
+        return mockRes({ payload: placeholders });
+      }
+
+      if (url.includes('icons.svg')) {
+        return mockRes({ payload: icons });
+      }
+
+      return null;
+    });
+  });
+
+  afterEach(() => {
+    clock.restore();
+    window.fetch = originalFetch;
+  });
+
+  describe('basic sanity tests', () => {
+    it('should have footer', async () => {
+      const footer = await createFullGlobalFooter({ waitForDecoration: true });
+      expect(footer).to.exist;
+      expect(document.querySelector('footer')).to.exist;
+    });
+
+    it('should have menu', async () => {
+      const footer = await createFullGlobalFooter({ waitForDecoration: true });
+
+      expect(footer).to.exist;
```

Comment:
Since you've got a test checking if this exists, it can IMO be skipped in other tests.